### ISSUE TITLE
Keyboard Maestro Integration

### ISSUE THREAD

**Issue by Enoch2090** [chenxuuu/chouxianghua#6]
Implemented the code into Keyboard Maestro. Now you can quickly convert your words into 🤡🐘 language by:
1. Select the desired words.
2. Run the action.
[抽象话.kmmacros.zip](https://github.com/chenxuuu/chouxianghua/files/4346813/kmmacros.zip)

**Comment by chenxuuu** [comment]
only for macOS?

**Comment by Enoch2090** [comment]
Yeah, because I don't use Windows very often……I suggest trying Quicker on Windows, which may help(But I'm not sure)
